frontend/src/modules/organization/pages/organization-common-page.vue

```diff
@@ -7,25 +7,39 @@
       :search-config="organizationSearchFilter"
       :saved-views-config="commonOrganizationSavedViews"
       hash="organizations"
+      class="pb-1"
       @fetch="fetch($event)"
     />
```

Comment:
We shouldn't use all organization filters: `:config="organizationFilters"`
List of filters is in the designs -> https://www.figma.com/design/CqK8KzwZFdPk7u3aypgT5a/2.-LFX-CM-%7C-Product---Admin?node-id=206-12020&t=qGHovB2TheRKggp4-4
![Screenshot 2024-08-23 at 11 30 22](https://github.com/user-attachments/assets/5c5f2c9e-00f4-4c48-b1d2-771407b319f8)